provisioning/Samples/service/ProvisioningAzureSasCredentialSample/ProvisioningAzureSasCredentialSample.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.Azure.Devices.Provisioning.Service;
+using Microsoft.Azure.Devices.Shared;
+using System;
+using System.Threading.Tasks;
+
+namespace ProvisioningAzureSasCredentialSample
+{
+    internal class ProvisioningAzureSasCredentialSample
+    {
+        private const string RegistrationIdPrefix = "sampleid-";
+        private const string TpmEndorsementKey =
+            "AToAAQALAAMAsgAgg3GXZ0SEs/gakMyNRqXXJP1S124GUgtk8qHaGzMUaaoABgCAAEMAEAgAAAAAAAEAxsj2gUS" +
+            "cTk1UjuioeTlfGYZrrimExB+bScH75adUMRIi2UOMxG1kw4y+9RW/IVoMl4e620VxZad0ARX2gUqVjYO7KPVt3d" +
+            "yKhZS3dkcvfBisBhP1XH9B33VqHG9SHnbnQXdBUaCgKAfxome8UmBKfe+naTsE5fkvjb/do3/dD6l4sGBwFCnKR" +
+            "dln4XpM03zLpoHFao8zOwt8l/uP3qUIxmCYv9A7m69Ms+5/pCkTu/rK4mRDsfhZ0QLfbzVI6zQFOKF/rwsfBtFe" +
+            "WlWtcuJMKlXdD8TXWElTzgh7JS4qhFzreL0c1mI0GCj+Aws0usZh7dLIVPnlgZcBhgy1SSDQMQ==";
+        private string RegistrationId;
+
+        ProvisioningServiceClient _provisioningServiceClient;
+        public ProvisioningAzureSasCredentialSample(ProvisioningServiceClient provisioningServiceClient)
+        {
+            _provisioningServiceClient = provisioningServiceClient;
+        }
+
+        public async Task RunSampleAsync()
+        {
+            await QueryIndividualEnrollmentsAsync().ConfigureAwait(false);
```

Comment:
As a rule, we don't include `ConfigureAwait(false)` in samples because we don't know the user's scenario, or if including this would cause problems in their app. This is guidance from the Azure SDK team.